public/javascripts/Admin/src/Admin.js

```diff
@@ -355,17 +355,13 @@ function Admin(_, $) {
                 $("#crosswalk-mean").html((crosswalkStats.mean).toFixed(2));
                 $("#crosswalk-std").html((crosswalkStats.std).toFixed(2));
 
-                var pedestrianSignalStats = getSummaryStats(pedestrianSignals, "severity");
```

Comment:
This line actually uses a variable called `pedestrianSignals` that is created above this section and needs to be deleted. Any time you're deleting something, make sure to check any variables/functions/HTML/CSS used in the deleted code, following it back to the source and deleting (after checking that it isn't used anywhere else)!

Reply:
Sorry I missed that. I have removed the variable and pushed.